packages/devel/libevent/package.mk

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0-only
```

Reply:
nah, I'd rather stick to -or-later (like we have in almost 50% of our package.mk files)